agilent_infiniium_scope.py

```diff
@@ -31,360 +31,389 @@
         """Get or set the trigger mode
+
         Must be "AUTO", "TRIG" (normal), or "SING" (single)
         """
         dev = self.selectedDevice(c)
         if mode is None:
             resp = yield dev.query('TRIG:SWE?')
         else:
             mode = mode.upper()
-            if mode not in ['AUTO','TRIG','SING']:
+            if mode not in ['AUTO', 'TRIG', 'SING']:
                 raise Exception('Mode must be "AUTO", "TRIG", or "SING".')
             else:
-                yield dev.write('TRIG:SWE '+mode)
+                yield dev.write('TRIG:SWE {}'.format(mode))
                 resp = yield dev.query('TRIG:SWE?')
         returnValue(resp)
 
-    @setting(150, side = 's', returns = ['s'])
-    def horiz_refpoint(self, c, side = None):
-        """Get or set the reference point for the horizontal position. Must be 'LEFT', 'CENT'er, or 'RIGH't.
+    @setting(150, side='s', returns=['s'])
+    def horiz_refpoint(self, c, side=None):
+        """Get or set the reference point for the horizontal position.
+
+        Must be 'LEFT', 'CENT'er, or 'RIGH't.
         """
         dev = self.selectedDevice(c)
         if side is None:
             resp = yield dev.query('TIM:REF?')
         else:
             side = side.upper()
-            if side not in ['LEFT','CENT','RIGH']:
+            if side not in ['LEFT', 'CENT', 'RIGH']:
                 raise Exception('Mode must be "LEFT", "CENT", or "RIGH".')
             else:
-                yield dev.write('TIM:REF '+side)
+                yield dev.write('TIM:REF {}'.format(side))
                 resp = yield dev.query('TIM:REF?')
         returnValue(resp)
 
-    @setting(151, position = 'v', returns = ['v'])
-    def horiz_position(self, c, position = None):
-        """Get or set the horizontal trigger position (wrt value from horiz_refpoint) in seconds.
+    @setting(151, position='v', returns=['v'])
+    def horiz_position(self, c, position=None):
+        """Get or set the horizontal trigger position.
+
+        With respect to value from horiz_refpoint, in seconds.
         """
         dev = self.selectedDevice(c)
         if position is None:
             resp = yield dev.query('TIM:POS?')
         else:
-            yield dev.write(('TIM:POS %f') %position)
+            yield dev.write('TIM:POS {}'.format(position))
             resp = yield dev.query('TIM:POS?')
         position = float(resp)
         returnValue(position)
 
-    @setting(152, scale = 'v', returns = ['v'])
-    def horiz_scale(self, c, scale = None):
+    @setting(152, scale='v', returns=['v'])
+    def horiz_scale(self, c, scale=None):
         """Get or set the horizontal scale
         """
         dev = self.selectedDevice(c)
         if scale is None:
             resp = yield dev.query('TIM:SCAL?')
         else:
-            scale = format(scale,'E')
-            yield dev.write('TIM:SCAL '+scale)
+            scale = format(scale, 'E')
+            yield dev.write('TIM:SCAL {}'.format(scale))
```

Comment:
Here again, just put formatting code in format string: `yield dev.write('TIM:SCAL {:E}'.format(scale))`